diff -ruN llvm-project-rocm-6.1.0.orig/amd/hipcc/hipcc-backward-compat.cmake llvm-project-rocm-6.1.0/amd/hipcc/hipcc-backward-compat.cmake
--- llvm-project-rocm-6.1.0.orig/amd/hipcc/hipcc-backward-compat.cmake	2025-01-31 10:51:48.032885869 +0100
+++ llvm-project-rocm-6.1.0/amd/hipcc/hipcc-backward-compat.cmake	2025-01-31 10:52:10.575251643 +0100
@@ -40,10 +40,10 @@
 create_binary_symlink()
 # TODO: Following has to modified if component based installation is required
 if (NOT WIN32)
-  install(DIRECTORY ${HIPCC_WRAPPER_BIN_DIR} DESTINATION hip)
-else()
   install(DIRECTORY ${HIPCC_WRAPPER_BIN_DIR} DESTINATION hip
           FILES_MATCHING
           PATTERN "*"
           PATTERN "*.bat" EXCLUDE )
+else()
+  install(DIRECTORY ${HIPCC_WRAPPER_BIN_DIR} DESTINATION hip)
 endif()
