From 062397da4331716c8f6ee7313b8d4c3fd6e33199 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Mon, 17 Jun 2024 11:46:22 -0500
Subject: [PATCH] Fix hip-config.cmake when it is not a symlink

---
 hipamd/hip-config.cmake.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hipamd/hip-config.cmake.in b/hipamd/hip-config.cmake.in
index 8b63fdc8b..532ccc058 100755
--- a/hipamd/hip-config.cmake.in
+++ b/hipamd/hip-config.cmake.in
@@ -93,6 +93,8 @@ endfunction()
 if(IS_SYMLINK ${CMAKE_CURRENT_LIST_FILE})
   get_filename_component(CONFIG_FILE_PATH "${CMAKE_CURRENT_LIST_FILE}" REALPATH)
   get_filename_component(PACKAGE_PREFIX_DIR "${CONFIG_FILE_PATH}/../../../../" ABSOLUTE)
+else()
+  get_filename_component(PACKAGE_PREFIX_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@/../" ABSOLUTE)
 endif()
 # end of TODO
 set(HIP_PACKAGE_PREFIX_DIR  ${PACKAGE_PREFIX_DIR})
-- 
2.44.0

